src/Functions/Public/resource-service/Set-vROResourceElement.ps1

```diff
@@ -0,0 +1,101 @@
+function Set-vROResourceElement {
+<#
+    .SYNOPSIS
+    Updates a resource element based off the resource ID.    
+
+    .DESCRIPTION
+    Updates a resource element based off the resource ID. 
+
+    .PARAMETER ResourceID
+    The ID of the Resource
+
+    .PARAMETER File
+    The resource file
+
+    .INPUTS
+    System.String
+
+    .OUTPUTS
+    None
+
+    .EXAMPLE
+    Get-ChildItem -Path "C:\Resources\$file" |Set-vROResourceElement -ResourceID $resource.Id -Confirm:$false
+
+#>
+[CmdletBinding(SupportsShouldProcess,ConfirmImpact="High")][OutputType('System.Management.Automation.PSObject')]
+
+    Param (
+
+    [parameter(Mandatory=$true)]
+    [ValidateNotNullOrEmpty()]
+    [String]$ResourceID,         
+    
+    [parameter(Mandatory=$true,ValueFromPipeline=$true,ValueFromPipelinebyPropertyName=$true)]
+    [ValidateNotNullOrEmpty()]
+    [String[]]$File
+
+    )
+
+    begin {
+
+        #Set Set Line Feed
+        $LF = "`r`n"
+    
+    }
+
+    process {
+
+        foreach ($FilePath in $File){
+
+            $FileInfo = [System.IO.FileInfo](Resolve-Path $FilePath).Path
+
+            try {
+
+                # --- Create the multi-part form
+                $Boundary = [guid]::NewGuid().ToString()
+                $FileBin = [System.IO.File]::ReadAllBytes($FileInfo.FullName)
+                $Encoding = [System.Text.Encoding]::GetEncoding("iso-8859-1")
+                $EncodedFile = $Encoding.GetString($FileBin)
+
+                $Form = (
+                    "--$($Boundary)",
+                    "Content-Disposition: form-data; name=`"file`"; filename=`"$($FileInfo.Name)`"",
+                    "Content-Type:application/octet-stream$($LF)",
+                    $EncodedFile,
+                    "--$($Boundary)--$($LF)"
+                ) -join $LF
+
+                $URI = "/vco/api/resources/$($ResourceID)"
+
+                # --- Set custom headers for the request
+                $Headers = @{
+                
+                    "Authorization" = "Basic $($Global:vROConnection.EncodedPassword)";
+                    "Accept" = "Application/json"
+                    "Accept-Encoding" = "gzip,deflate,sdch";
+                    "Content-Type" = "multipart/form-data; boundary=$($Boundary)"
+                }
+
+                if ($PSCmdlet.ShouldProcess($FileInfo.FullName)){
```

Comment:
Please replace $FileInfo.FullName with $Id so that during a whatif the impact message will be 

```What if: Performing the operation "Set-vROResourceElement" on target "5adbda5e-5c35-4cb8-9ff7-ce165e7004c8"```

and not 

```What if: Performing the operation "Set-vROResourceElement" on target "C:\Users\a605603\Documents\test.txt".```

since different to the import function, we are make the changes on an existing item.